ports/osgearth/portfile.cmake

```
@@ -1,17 +1,12 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO gwaldron/osgearth
-    REF "osgearth-${VERSION}"
-    SHA512 f65c31922bebcbf722474a047dc29c8c1ceec9c037b0704811af2627fc2d0a124b6e95888e7d3b9b0e5acc146a88ebf8669e3f864a75a91751c3a4571d05a630
+    REF 1a4344609c0e58aa3e6b1a9cf7a6f82984337e6c
```

With vcpkg.json updated, you can restore
```suggestion
    REF "osgearth-${VERSION}"
```

---

I see that osgearth 3.7 isn't release yet. Then this PR should be a draft until the git tag is available and immutable.